custom/icds_reports/reports/immunization_coverage_data.py

```diff
@@ -183,10 +183,13 @@ def get_immunization_coverage_data_chart(domain, config, loc_level, show_test=Fa
 
     config['month__range'] = (three_before, month)
```

Comment:
Is this PR raised Before your Immunization change. I cant see those changes here.

Reply:
i will merge master

Reply:
done